deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -0,0 +1,21 @@
+import re
```

Comment:
Here you should add:
```
# Copyright (C) 2015-2020, Wazuh Inc.
# Created by Wazuh, Inc. <info@wazuh.com>.
# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
```

Reply:
Done in 98f3a762b6ad89afab9ebb986feb5555545c30fe